### PR TITLE
Sidebar Styling

### PR DESCRIPTION
**Pull request by christinem**
This PR fixes some styling for the sidebar. 

If there are any other stylistic suggestions that people have, please let me know!

### CODE REVIEW

File: hs/GraphResponse.hs [hunk 1]
```diff
@@ -78,8 +78,8 @@ sideBar = do
             H.p ! A.id "web" ! A.class_ "focus" $ "Web Technologies"
             H.div ! A.id "web-details" ! A.class_ "details" $ ""
         H.div ! A.id "graphs" $ ""
-    H.div ! A.id "sidebar-button" $ ""
-
+    H.div ! A.id "sidebar-button" $ do
```
Comment (Ian-Stewart-Binks) [inline hs/GraphResponse.hs]: This `do` is unnecessary.

File: hs/GraphResponse.hs [hunk 2]
```diff
@@ -78,8 +78,8 @@ sideBar = do
...
+        H.img ! A.id "sidebar-icon" ! A.src "static/res/ico/sidebar.png"
```
Comment (Ian-Stewart-Binks) [inline hs/GraphResponse.hs]: There are a lot of blank lines at the end of this file.

File: hs/GraphResponse.hs (hunk 1 above)
Comment (christinem) [inline hs/GraphResponse.hs]: Fixed.

File: hs/GraphResponse.hs (hunk 2 above)
Comment (christinem) [inline hs/GraphResponse.hs]: Fixed.

**Comment by david-yz-liu** [comment]
The icon should be a little smaller, but vertically centered in the "button".

**Comment by christinem** [comment]
@david-yz-liu Done!

**Pull request merged**